src/app/components/core/configuration-manager/configuration-group/configuration-group.component.html

```diff
@@ -15,7 +15,7 @@
       <div [id]="'nav_contents_'+from + '_' + sourceName" class="nav">
         <div class="tabs" *ngIf="from !== 'config-manager'"
           [ngClass]="{'is-fullwidth': sourceName && groupTabs?.length > 1 && from !== 'notifications' && developerTabState(groupTabs), 'is-boxed': !sourceName}">
-          <ul>
+          <ul class="group-nav-list">
```

Comment:
remove this unwanted class.

Reply:
This is not an unwanted class. This has been added intentionally.
Previously, there were double borders under the tabs and a single border in areas without tabs, due to which thickness of this line was not consistent. It should be a single consistent border covering the entire area below the tabs.